x-pack/plugins/spaces/public/management/edit_space/manage_space_page.tsx

```diff
@@ -110,46 +111,48 @@ export class ManageSpacePage extends Component<Props, State> {
   }
 
   public render() {
-    const content = this.state.isLoading ? this.getLoadingIndicator() : this.getForm();
+    if (!this.props.capabilities.spaces.manage) {
+      return (
+        <EuiPageContent verticalPosition="center" horizontalPosition="center" color="danger">
+          <UnauthorizedPrompt />
+        </EuiPageContent>
+      );
+    }
+
+    if (this.state.isLoading) {
+      return this.getLoadingIndicator();
+    }
 
     return (
       <Fragment>
-        <EuiPageContentBody>{content}</EuiPageContentBody>
+        <EuiPageHeader
+          bottomBorder
+          pageTitle={this.getTitle()}
+          description={getSpacesFeatureDescription()}
+        />
+        <EuiSpacer size="l" />
+
+        {this.getForm()}
       </Fragment>
     );
   }
 
   public getLoadingIndicator = () => (
-    <div>
-      <EuiLoadingSpinner size={'xl'} />{' '}
-      <EuiTitle>
-        <h1>Loading...</h1>
-      </EuiTitle>
-    </div>
+    <EuiPageContent verticalPosition="center" horizontalPosition="center" color="subdued">
```

Comment:
This loading screen isn't getting properly centered. My guess is there's an extra wrapping div somewhere that needs to be removed so that EuiPageContent is always the first descendent of the page template.

<img width="576" alt="Screen Shot 2021-06-10 at 14 34 53 PM" src="https://user-images.githubusercontent.com/549577/121578860-25042380-c9f9-11eb-9722-668e080e7fa8.png">